src/hotspot/share/runtime/deoptimization.cpp

```diff
@@ -842,13 +842,24 @@ void Deoptimization::unwind_callee_save_values(frame* f, vframeArray* vframe_arr
 }
 
 #ifndef PRODUCT
+// Return true if the execution after the provided bytecode continues at the
```

Comment:
```suggestion
// Return true if the execution after the provided bytecode can continue at the
```
Nit: Because a `cmp_if` may or may not continue with the next bci.

Reply:
I looked into that and both basic blocks following the `if_cmp` will always be reachable as far as `bb->is_reachable()` is concerned. The bytecode verification code of the `if_cmp` to check that the stack depth on both the then and else branch are the same requires both basic blocks to be reachable.

Reply:
Yes, the function name and comment aren't as precise as they could be.  "parsing" might have been a better word than "execution", for example.  But I think the function name is OK as it is.  We use variants of this naming elsewhere for exactly the same purpose:
https://github.com/openjdk/jdk/blob/e7ce661adb01fba4bb690d51cc2858c822008654/src/hotspot/share/oops/generateOopMap.cpp#L1159